docker/1.6.0/py2/Dockerfile.cpu

```diff
@@ -32,8 +32,10 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && ln -s $(which python) /usr/local/bin/python
 
+# MXNet requires pip 19.3.1 due to being backwards compatible
```

Comment:
This is wrong. pip 20.0.2 supports installing mxnet again. Reference https://github.com/pypa/pip/issues/7626

Reply:
@codewithsk